src/drivers/intel/dmic/dmic.c

```diff
@@ -1183,116 +217,49 @@ static int dmic_set_config(struct dai *dai, struct ipc_config_dai *common_config
 			dmic_prm[i] = (struct sof_ipc_dai_dmic_params *)
 				((uint8_t *)uncached_dmic_prm[i - 1] + size);
```

Comment:
Uncached use here is not consistent with the approach

Reply:
my expectation is that the DMIC driver should be using ```SOF_MEM_ZONE_RUNTIME_SHARED``` for all memory allocations.

Reply:
Thanks, it seems that they are all correct.